CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Changelog
 
-## Unreleased
-- Fix #400 Docker SELinux context not set properly when building vagrant box @praveenkumar
+## v2.2.0 June 9, 2016
+- Update variant information @praveenkumar
+- Can not change securityContext in k8s @praveenkumar
+- Fix #422 Failed to get pwuid struct: user: unknown userid @praveenkumar
+- Reduce vagrant box size #176 @praveenkumar
+- rhel-k8s Vagrantfile: fix in getting credentials @optak
+- Changes for Release 2.2.0 @LalatenduMohanty
+- Add openshift tag information to kickstart instead in sccli/openshift_options @praveenkumar
+- Fix #415 add yum clean all step to kickstart file to remove yum local cache @praveenkumar
+- Fix #414 kube-apiservice shouldn't be part of kickstart file @praveenkumar
+- Remove Public_Host variable from OpenShift Vagrantfile @praveenkumar
+- Fix 407 for "Error: dev/null: No such file or directory" @praveenkumar
```

Comment:
@budhrg will fix it. That was missing in the original commit message I guess. Do we need this much details here?